bin/concat_tsv.py

```diff
@@ -28,35 +30,39 @@ def parse_args(args=None):
     return parser.parse_args()
 
 
+# Peptide Predictions remain unsorted, otherwise full dataframe needs to be loaded to memory, which would be computational very intensive.
+
 def main(args=None):
     args = parse_args(args)
 
-    first_header = pd.DataFrame().columns
-    for i, filename in enumerate(args.input):
-        print("Processing file: ", filename, flush=True)
-
-        # Read input file chunk-wise
-        with pd.read_csv(filename, sep="\t", chunksize=args.chunk_size) as reader:
-            for j, tsv_chunk in enumerate(reader):
-                print(" Chunk: ", j, flush=True)
-                if i == 0 and j == 0:
-                    first_header = tsv_chunk.columns
-                    print("Header: ", first_header.tolist(), flush=True)
-                    tsv_chunk.to_csv(args.output, mode="w", sep="\t", index=False, header=True)
-                else:
-                    if j == 0:
-                        # Check if header of subsequent input files match header of first input file
-                        # (column order must be the same)
-                        if tsv_chunk.columns.tolist() != first_header.tolist():
-                            print(
-                                "ERROR - header of input file",
-                                filename,
-                                "does not match the header of the first input file!",
-                                file=sys.stderr,
-                            )
-                            sys.exit(1)
-
-                    tsv_chunk.to_csv(args.output, mode="a", sep="\t", index=False, header=False)
+    with TextIOWrapper(GzipFile(args.output, 'w', mtime=0), encoding='utf-8') as outfile:
```

Comment:
Could you explain why you added this?

Reply:
The gzips have timestamps, which are added automatically. They can be changed using the `mtime` parameter, but not completely disabled. If the `gzips.open()` function is used, the timestamp is defaulted to the current time as there is currently no option to access the `mtime` parameter. All in all this is a workaround to open a gzip file with `mtime=0`, which does not disable the timestamp but keeps it constant at 0.

Reply:
Is this done by others as well? 
I found this https://nfcore.slack.com/archives/CJRH30T6V/p1702934790359819?thread_ts=1702554523.006219&cid=CJRH30T6V 
sounds as if it could be compared only based on the content without timestamp

Reply:
Thats odd, because I actually had that problem, the files gzpped files were exactly the same, but md5sum comparison of the .gz failed! Also I think if gzip is used as standalone it behaves differently compared to within python!

Reply:
hmm... but the `.gz` structure/header shouldn't be different. Did you see https://github.com/askimed/nf-test/pull/144?

Reply:
other tools also create  gzip files I would guess

Reply:
This is interesting, I will try again! Sure gzip is used often, but usually outside of the tools/scripts as standalone.

Reply:
Well absolutely my bad, I have checked the md5sums always using standalone 'md5sum'!! I have reverted the workaround!